omod/src/main/java/org/openmrs/calculation/web/controller/CalculationRegistrationFormController.java

```diff
@@ -136,6 +138,6 @@ public String purgeCalculationRegistration(@ModelAttribute(value = "calculationR
 	
 	private void updateSessionMessage(WebRequest request, String code, Object... args) {
 		String msg = Context.getMessageSourceService().getMessage(code, args, Context.getLocale());
-		request.setAttribute(WebConstants.OPENMRS_MSG_ATTR, msg, WebRequest.SCOPE_SESSION);
+		request.setAttribute(WebConstants.OPENMRS_MSG_ATTR, Encode.forHtml(msg), WebRequest.SCOPE_SESSION);

```

Comment:
@anupamdevulapalli I've been mulling this over and I think the best workaround is to remove the encoding here and all OWASP encoder dependencies and then delete the last argument in the call to `updateSessionMessage` on line 126. 

That way, there's no user-controlled input in the update message at all, and we can remove the OWASP dependencies to avoid the issues @dkayiwa brought up.

Not the most elegant solution, but should be pretty straightforward.

Reply:
@isears I've made the changes and please let me know those look fine

Reply:
@anupamdevulapalli that looks good except return the `updateSessionMessage` function to its original state, so it should include the lines.

```
String msg = Context.getMessageSourceService().getMessage(code, args, Context.getLocale());
request.setAttribute(WebConstants.OPENMRS_MSG_ATTR, msg, WebRequest.SCOPE_SESSION);
```

Thanks for your patience!

Reply:
@isears I added the lines back. But the banner on the top of the screen displays "Saved {0} Calculation" whenever a user saves a new calculation like [this](https://drive.google.com/file/d/1SNHKn4pKKrCsdQAZDp4ZoYOo-COjeIoJ/view?usp=sharing).

Reply:
@isears @dkayiwa I edited the message properties file to include a new message which does not depend on the name of the calculation. Please let me know if this works.

Thank you for helping me out with this issue!